packages/relay/src/formatters.ts

```diff
@@ -79,6 +79,21 @@ const formatTransactionId = (transactionId: string): string | null => {
   return `${payer}-${timestamp}`;
 };
 
+/**
+ * Formats a transaction object by moving input to data, if data is absent.
+ * @param {Object} transaction - The transaction object to be formatted.
+ */
+const formatTransaction = (transaction: any) => {
+  /**
+   * If the transaction has an input property with a length greater than zero and does not have a data property,
+   * assign the value of transaction.input to transaction.data and remove the input property.
+   */
+  if (transaction.input.length > 0 && !transaction.data) {
```

Comment:
Great. I fyou can add a bit more info. If I recall correctly it's something around the execution apis changes the spec or some EVM clients use one field and others use the other.
I just want people to have context on why it could be either option

Reply:
I've udpated the description of the PR. And for more info, before this fix, when we sent this to the mirror, it returns `0x`, which is invalid and this is why we return 400k gas. The reason why it returns `0x`, is because we initiate gas estimation for contract create, but our tx object does not have a data field.